docs/contributor_guide/developer_call_notes/2020_09_17.rst

```diff
@@ -0,0 +1,69 @@
+Developer Call September 17, 2020
+---------------------------------
+
+These are collaborative notes.
+
+Attendees:
+- `Miro Dudik <https://github.com/MiroDudik>`_
+- `Richard Edgar <https://github.com/riedgar-ms>`_
+- `Adrin Jalali <https://github.com/adrinjalali>`_
+- `Roman Lutz <https://github.com/romanlutz>`_
+- `Andreas Mueller <https://github.com/amueller>`_
+- `Kevin Robinson <https://github.com/kevinrobinson>`_
+- `Mehrnoosh Sameki <https://github.com/mesameki>`_
+- `Hilde Weerts <https://github.com/hildeweerts>`_
+
+
+Agenda:
+
+- Datasets module: keep or remove?
+
+    - not part of a release yet (so removing would be easy)
+    - reasons for including are technical:
+
+        - show how to use a function (create "minimal examples")
+        - benchmark algorithms, e.g., how well they optimize
+          fairness/performance trade-off under specific quantitative
+          definitions of fairness and performance
+    
+    - but including datasets means we should address sociotechnical aspects
+    - we don't add real functionality, just use functionality from
+      `sklearn.datasets`, namely `fetch_openml`
+
+        - potential value prop: we can return sensitive features in addition
+          to X, y
+
+    - e.g., https://github.com/fairlearn/fairlearn/pull/519
+
+    - other options (other than keep/remove):
+
+        - don't spend more time on it
+        - remove module, but move content into examples
+
+    - Roman to summarize this info in an issue and get Vincent's feedback
+
+- Should we restructure our weekly sessions?
+
+    - current situation:
+
+        - developer call once a month (2nd Thursday of each month, today being
+          an exception)
+        - sociotechnical deep dive on all other Thursdays
+
+    - changes to this schedule are possible
+    - Feedback: get agenda out at least a couple of days before the call!
+
+- Metrics API progress
+
+    - Proposal: https://github.com/fairlearn/fairlearn-proposals/pull/15 
+      --> Richard started implementation
```

Comment:
Want to add a link to my fork/branch:
https://github.com/riedgar-ms/fairlearn/blob/riedgar-ms/return-of-the-GMR/test/unit/metrics/experimental/sample_code.py